clang/test/Preprocessor/riscv-target-features.c

```diff
@@ -1220,3 +1220,15 @@
 // RUN: -march=rv64i_zve32x_zvkt1p0 -x c -E -dM %s \
 // RUN: -o - | FileCheck --check-prefix=CHECK-ZVKT-EXT %s
 // CHECK-ZVKT-EXT: __riscv_zvkt 1000000{{$}}
+
+// RUN: %clang -target riscv32-unknown-linux-gnu -march=rv32i -x c -E -dM %s \
```

Comment:
Use `--target=` for new tests. `-target ` has been deprecated since 3.4.

Prefer `%clang_cc1` for tests outside of test/Driver.

Reply:
To be fair, the tests in this file are currently %clang. I would argue we should be consistent, which either means %clang here or someone converting the others to %clang_cc1 (so long as those still make sense as cc1 tests)

Reply:
Should we update this test file to use `--target=` for all test cases in another patch? Getting a review comment for copy and pasting from an existing test case is frustrating.